src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json

```diff
@@ -0,0 +1,64 @@
+{
+  "description": "RIA-2999 Appellant FTPA Application is allowed  after the respondent application already dismissed",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "Judge",
+    "input": {
+      "id": 1007,
+      "eventId": "residentJudgeFtpaDecision",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ftpaAppellantDecisionOutcomeType": "allowed",
+          "ftpaRespondentDecisionOutcomeType": "dismissed",
+          "ftpaApplicantType": "appellant",
+          "ftpaApplicantType": "respondent",
+          "ariaListingReference": "987654321",
+          "notificationsSent": [
+            {
+              "id": "1007_FTPA_APPLICATION_DECISION_HOME_OFFICE_RESPONDENT",
+              "value": "34e146c9-6050-4894-a64e-acd461a5f934"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "notificationsSent": []
+      }
+    },
+    "notifications": [
+      {
+        "reference": "1007_FTPA_APPLICATION_DECISION_HOME_OFFICE_APPELLANT",
+        "recipient": "{$respondentEmailAddresses.respondentEvidenceDirection}",
+        "subject": "Immigration and Asylum appeal: application to appeal to Upper Tribunal",
+        "body": [
+          "PA/12345/2019",
+          "A1234567",
+          "987654321",
+          "Talha Awan"
+        ]
+      }
+    ],
+    "notifications": [
```

Comment:
These functional tests will always pass, regardless of the recipient address.

The `"notifications": [` needs to be removed as was done in RIA-1432.

As it stands, with more than one "notifications" node, only one is passed to assert function, which may give you a false positive functional test. These changes will need to made to the rest of the functional tests too.

Reply:
Fixed